Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -98,6 +101,11 @@ namespace ROS2
         {
             serialize->Class<ROS2SpawnerComponent, ROS2SpawnerComponentBase>()->Version(1);
         }
+
+        if (AZ::BehaviorContext* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
+        {
+            behaviorContext->EBus<SpawnerNotificationBus>("SpawnerNotificationBus")->Handler<SpawnerNotificationsBusHandler>();
```

Comment:
One more comment - since we are calling the component `ROS2SpawnerComponent`, I would suggest keeping `ROS2` within the name, so we can easily differentiate it from the built in spawner notifications.
```suggestion
            behaviorContext->EBus<SpawnerNotificationBus>("ROS2SpawnerNotificationBus")->Handler<SpawnerNotificationsBusHandler>();
```